content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -191,7 +191,8 @@ For a reference to old feature gates that are removed, please refer to
 | `ServiceNodePortStaticSubrange` | `false` | Alpha | 1.27 | |
 | `SizeMemoryBackedVolumes` | `false` | Alpha | 1.20 | 1.21 |
 | `SizeMemoryBackedVolumes` | `true` | Beta | 1.22 | |
-| `StatefulSetAutoDeletePVC` | `false` | Alpha | 1.22 | |
+| `StatefulSetAutoDeletePVC` | `false` | Alpha | 1.22 | 1.27 |
+| `StatefulSetAutoDeletePVC` | `false` | Beta | 1.28 | |
```

Comment:
are you serious? We are documenting something for 1.28 now?

Reply:
No, I just got confused about what release cycle we're on... fixed